src/test/java/Homework27.java

```diff
@@ -0,0 +1,56 @@
+import Pages.HomePage;
+import Pages.LoginPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework27 extends BaseTest {
+    @Test(enabled = true, priority = 0, description = "Login with valid credentials")
+
+    public void loginWithValidCredentials() {
+        LoginPage loginPage = new LoginPage(getThreadLocal());
+        HomePage homePage = new HomePage(getThreadLocal());
+
+        loginPage.login();
+        Assert.assertTrue(homePage.getUserAvatar().isDisplayed());
+    }
+
+    @Test(enabled = true, priority = 1, description = "create a playlist")
+    public void createNewPlaylist() {
+        LoginPage loginPage = new LoginPage(getThreadLocal());
+        HomePage homePage = new HomePage(getThreadLocal());
+
+        loginPage.login();
+        homePage.createPlaylist();
+        Assert.assertTrue(HomePage.newPlaylistIsDisplayed());
+    }
+
+    @Test(enabled = true, priority = 2, description = " add a song to playlist")
+    public void addSongToExistingPlaylist() throws InterruptedException {
```

Comment:
Please, **delete** `throws InterruptedException`

**Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.**

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.